orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -966,6 +966,43 @@ def run_step(model, input):
         _test_helpers.assert_values_are_close(ort_prediction, pt_prediction)
         _test_helpers.assert_values_are_close(ort_input.grad, pt_input.grad)
 
+# Since multinomial is a generator function, we do not have to test for gradient
+# Two consecutive calls on the torch.multinomail on a probability distribution with more 
```

Comment:
calling torch.manual_seed will reset the torch generator, the resulting generated sequence will be the same. 


>>> torch.manual_seed(42)
<torch._C.Generator object at 0x7f1d20c0f990>
>>> torch.multinomial(x,2,True)
tensor([0, 0])
>>> torch.multinomial(x,2,True)
tensor([1, 0])
>>> torch.multinomial(x,2,True)
tensor([2, 2])
>>> torch.multinomial(x,2,True)
tensor([3, 3])
>>> torch.manual_seed(42)
<torch._C.Generator object at 0x7f1d20c0f990>
>>> torch.multinomial(x,2,True)
tensor([0, 0])
>>> torch.multinomial(x,2,True)
tensor([1, 0])
>>> torch.multinomial(x,2,True)
tensor([2, 2])
>>> torch.multinomial(x,2,True)
tensor([3, 3])

Reply:
that is right, are you suggesting we reset the seed between calls to the ort and torch models, and still pass random data?

Reply:
yes